.github/CODEOWNERS

```diff
@@ -128,7 +128,7 @@
 /Makefile.trace                         @DataDog/agent-apm
 
 /omnibus/                               @DataDog/agent-platform
-/omnibus/config/software/datadog-agent*.rb                @Datadog/agent-core @DataDog/agent-platform
```

Comment:
I think we should remove this line entirely, having `/omnibus/config/software/datadog-agent*.rb` owned by agent-platform like the rest of `/omnibus/` makes sense to me